examples/uct_hello_world.c

```diff
@@ -253,15 +253,13 @@ static ucs_status_t init_iface(char *dev_name, char *tl_name,
     params.field_mask           = UCT_IFACE_PARAM_FIELD_OPEN_MODE   |
                                   UCT_IFACE_PARAM_FIELD_DEVICE      |
                                   UCT_IFACE_PARAM_FIELD_STATS_ROOT  |
-                                  UCT_IFACE_PARAM_FIELD_RX_HEADROOM |
-                                  UCT_IFACE_PARAM_FIELD_CPU_MASK;
+                                  UCT_IFACE_PARAM_FIELD_RX_HEADROOM;
```

Comment:
IMO better not remove it from hello world - since it's an example

Reply:
Done.